docs/sources/api/_index.md

```diff
@@ -609,6 +610,21 @@ running Loki on Kubernetes, `/ready` can be used as a readiness probe.
 
 In microservices mode, the `/ready` endpoint is exposed by all components.
 
+## Change log level at runtime
+
+```
+GET /log_level
+POST /log_level
+```
+
+`/log_level` returns the log level in case of `GET` and allows to change the log level at runtime in case of `POST`.
```

Comment:
```suggestion
`/log_level` a `GET` returns the current log level and a `POST` lets you change the log level of a Loki process at runtime. This can be useful for accessing debugging information during an incident. Caution should be used when running  at the `debug` log level, as this produces a large volume of data.
```

Reply:
@JStickler Thank you for the suggestion - I was struggling to write the description myself and this makes a lot more sense.

I accepted your suggestion already, but would it be better to remove the endpoint from the beginning? Or use a colon to separate the endpoint from the description. What would you prefer?

Reply:
>would it be better to remove the endpoint from the beginning? Or use a colon to separate the endpoint from the description. 

@Indransh Looking over the rest of the topic, it looks like most of the entries have the endpoint at the beginning of the description, and do not have a colon to separate the endpoint from the description, so lets follow that convention so that things look consistent.

Reply:
Thanks, then lets keep it consistent. I pushed a small fix (remove extra space) and re-requested your review.

Resolving this thread since the original suggestion was accepted.